code/workspaces/infrastructure-api/resources/vscode.deployment.template.yml

```diff
@@ -0,0 +1,106 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: {{ name }}
+  name: {{ name }}
+spec:
+  selector:
+    matchLabels:
+      name: {{ name }}
+  template:
+    metadata:
+      labels:
+        name: {{ name }}
+        user-pod: {{ type }}
+    spec:
+      securityContext:
+        runAsUser: 0
```

Comment:
I think we should set this to `1000` which is the UID used by all other notebooks in DataLabs just so we don't get permission errors.

Reply:
that's done - thanks for explaining